style.css

```diff
@@ -0,0 +1,18 @@
+body{
+    margin: 0;
+}
+*{
+    padding: 0;
+    margin: 0;
+    list-style: none;
+    text-decoration: none;
+    box-sizing: border-box;
+    outline: none;
+    border: none;
+    vertical-align: top;
+}
+.header{
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
```

Comment:
add container and row and column comman property

Reply:
ok